torch/_guards.py

```diff
@@ -708,15 +710,24 @@ def current_frame(frame_summary):
     @contextlib.contextmanager
     def report_output_strides():
```

Comment:
Dont actually need these changes anymore.. Maybe cleaner this way? either way, can delete